frontend/__tests__/services/wsaddress-service.server.test.ts

```diff
@@ -0,0 +1,34 @@
+/* eslint @typescript-eslint/no-unused-vars: ["error", { "varsIgnorePattern": "^_" }] */
```

Comment:
This rule configuration isn't required for this suite of tests.

Reply:
Removed outdated config